cluster/saltbase/salt/docker/init.sls

```diff
@@ -104,9 +104,24 @@ lxc-docker-{{ docker_ver }}:
     - sources:
       - lxc-docker-{{ docker_ver }}: /var/cache/docker-install/{{ deb }}
 
+docker_load_kube_apiserver:
```

Comment:
These functions are stuck in a if/else block that is only called if you are not running in a Red Hat based linux distro, so they are never called.  I am testing this by moving it to the top of the function.